src/components/choice-tabs/ChoiceTabs.scss

```
@@ -0,0 +1,268 @@
+  gap: 6px;
+  margin-top: auto;
+  padding-top: 12px;
+  color: var(--home-features-item-desc-color, rgba(17, 17, 19, 0.65));
+}
+
+.choice-tabs__platform-icon {
+  width: 13px !important;
+  height: 13px !important;
+  min-width: 13px;
+  min-height: 13px;
+  background-color: currentColor;
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)